src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -204,12 +207,19 @@ class Repository(
     return offender
   }
 
-  fun lookupActivity(id: Long): CourseActivity = activityRepository.findById(id).orElseThrow().also {
+  fun lookupProgramService(id: Long): ProgramService = programServiceRepository.findByIdOrNull(id)!!
+  fun lookupOffenderProgramProfile(id: Long): OffenderProgramProfile =
+    offenderProgramProfileRepository.findByIdOrNull(id)!!
+
+  fun lookupActivity(id: Long): CourseActivity = activityRepository.findByIdOrNull(id)!!.also {
     it.payRates?.size
   }
```

Comment:
What's the size for? It doesn't seem to do anything...

Reply:
Its 'hydrating' the list